packages/mask/src/plugins/Trader/trader/useTradeContext.ts

```diff
@@ -190,6 +192,17 @@ export function useTradeContext(tradeProvider: TradeProvider) {
                     TYPE: tradeProvider,
                     ROUTER_CONTRACT_ADDRESS: DEX_TRADE.OPENOCEAN_EXCHANGE_PROXY_ADDRESS,
                 }
+            case TradeProvider.WOOFI:
+                return {
+                    TYPE: tradeProvider,
+                    GRAPH_API: DEX_TRADE.WOOFI_THEGRAPH,
+                    INIT_CODE_HASH: DEX_TRADE.WOOFI_INIT_CODE_HASH,
+                    ROUTER_CONTRACT_ADDRESS: DEX_TRADE.WOOFI_ROUTER_ADDRESS,
+                    FACTORY_CONTRACT_ADDRESS: DEX_TRADE.WOOFI_FACTORY_ADDRESS,
+                    AGAINST_TOKENS: WOOFI_BASE_AGAINST_TOKENS,
+                    ADDITIONAL_TOKENS: {},
+                    CUSTOM_TOKENS: WOOFI_CUSTOM_BASES,
```

Comment:
Those fields are not required for a API-based trader.

```suggestion
```

Reply:
I have removed GRAPH_API, INIT_CODE_HASH, and FACTORY_CONTRACT_ADDRESS and keeping the rest because the swap is performed by calling ROUTER_CONTRACT_ADDRESS contract directly